packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -43,6 +43,8 @@ async function main() {
         mkdir("-p", "tests/testProject/widgets");
         cp("-rf", `dist/${widgetVersion}/*.mpk`, "tests/testProject/widgets/");
     }
+    // Downloading test project
+    await unzipTestProject();
```

Comment:
Maybe we need to introduce a feature flag like the `--mx-version` to prevent to download the test project. If external users have their own test project this line will always override their projects. Could be something like `--ignore-testproject-download` or any other idea :)